docs/Installation.md

```diff
@@ -1,8 +1,121 @@
+
+   ```bash
+    oc login -u <user> -p <password> --server=<oc_api_url>
+   ```
+
+2. Export required (and recommended) environment variables from [default.env](default.env). Copy the file (`cp default.env user.env`), edit the required variables and source it (`source user.env`).
+
+3. Install dependencies:
+
+``` bash
+# Install dependencies
+$ go mod tidy
+# or go mod tidy -compat=1.19
+# Copy the dependencies to vendor folder
+$ go mod vendor
+```
+
+1. Install RHTAP in e2e mode. By default the installation script will use the `redhat-appstudio-qe` GitHub organization for pushing changes to `infra-deployments` repository.
+
+**It is recommended to use your fork of [infra-deployments repo](https://github.com/redhat-appstudio/infra-deployments) in your GitHub org instead** - you can change the GitHub organization with environment variable `export MY_GITHUB_ORG=<name-of-your-github-org>`.
+
+Some tests could require you have a Github App created in order to test Component builds via Pipelines as Code.
+Such tests are [rhtap-demo](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go) and [build](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go).
+
+In this case, before you bootstrap a cluster, make sure you [created a Github App for your GitHub account](https://github.com/settings/apps). Fill in following details:
+* **GitHub App name**: unique app name
+* **Homepage URL**: some dummy URL (like https://example.com)
+* **Webhook**: mark as active and put some dummy URL to the **Webhook URL** field
+* **Permissions** and **Subscribe to events**: refer to [the guide in PaC documentation](https://pipelinesascode.com/docs/install/github_apps/#setup-manually)
+* **Where can this GitHub App be installed?**: Any account
+
+Hit create, make a note of the **App ID** and navigate to the **Private keys** section where you generate a private key that gets downloaded automatically. Then export following environment variables (or if you're using [.env file](default.env), update values of following variables):
+
+```
+export E2E_PAC_GITHUB_APP_ID='<YOUR_APP_ID>'
+
+export E2E_PAC_GITHUB_APP_PRIVATE_KEY=$(base64 < /PATH/TO/YOUR/DOWNLOADED/PRIVATE_KEY.pem)
+```
+
+Navigate back to [your GitHub App](https://github.com/settings/apps), select **Install App** and select your GitHub org (the one that you're using in `MY_GITHUB_ORG` env var). Feel free to install it to all repositories of that organization or the **forked repositories** currently used by [rhtap-demo](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go) and [build](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go) tests ([`hacbs-test-project`](https://github.com/redhat-appstudio-qe/hacbs-test-project) and [`devfile-sample-hello-world`](https://github.com/redhat-appstudio-qe/devfile-sample-hello-world)).
```

Comment:
Can we visually differentiate this section in some way (by adding indentation, or moving it under a new heading), so that it becomes clear that these instructions are only for rhtap-demo and build test suite -- I know it's already written there, I think it might look more readable if we separate it visually as well so that generally, right after a person runs `go mod vendor` they would know that they need to run `make local/test/e2e`. Wdyt?